src/common/libsubprocess/test/test_multi_echo.c

```diff
@@ -36,19 +45,57 @@ main (int argc, char *argv[])
         }
     }
 
+    if ((out + err) == 0) {
+        fprintf (stderr, "must specify a way to output");
```

Comment:
Missing newline on error message.  Maybe "-O and/or -E are required" would be a more useful message.

Reply:
Newline is still missing...

Reply:
doh!  also fixed in `test_echo`.